Server/mods/deathmatch/logic/CConsoleCommands.cpp

```diff
@@ -1240,60 +1240,45 @@ bool CConsoleCommands::WhoIs(CConsole* pConsole, const char* szArguments, CClien
 
 bool CConsoleCommands::DebugScript(CConsole* pConsole, const char* szArguments, CClient* pClient, CClient* pEchoClient)
```

Comment:
If you have already decided to rewrite, it would be great to get rid of the Hungarian notation.

Reply:
OK, i've done it. Lmk what you think